org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "opent2t-translator-wink-thermostat",
+  "version": "1.0.0",
+  "description": "Wink Thermostat translator",
+  "author": "OpenT2T",
+  "license": "MIT",
+  "scripts": {
+    "test": "ava"
+  },
+      "ava": {
+      "files": [
+        "tests/*.js",
+        "!**/deviceconfig.js"
+      ]
+    },
+  "dependencies": {
+    "delay": "^1.3.1",
+    "opent2t": "*",
+    "opent2t-onboarding-winkhub": "*",
+    "opent2t-translator-helper-wink": "1.0.0",
+    "q": "1.4.1"
+  },
+  "devDependencies": {
+    "ava": "^0.15.2"
+  },
+  "opent2t": {
+    "interfaces": {
+      "org.OpenT2T.Sample.SuperPopular.Thermostat": {
+        "description": "OpenT2T SuperPopular Thermostat interface"
+      }
+    },
+    "translators": {
+      "TranslatorA": {
```

Comment:
I guess this is OK for now, but it goes against the design principles that I had in mind for how we use modules. And having all translators named "thingTranslator.js" won't work when we transition to the new directory structure which can have multiple translators in a package.

---
In reply to: [73766739](https://github.com/openT2T/translators/pull/37#discussion_r73766739) [](ancestors = 73766739)